extensions/blocks/rating-star/editor.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import { registerBlockType } from '@wordpress/blocks';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+import save from './save';
+import { StarIcon, StarBlockIcon } from './icon';
+import './editor.scss';
+import './style.scss';
+
+registerBlockType( 'jetpack/rating-star', {
+	title: 'Star Rating',
+	description: __( 'Rate movies, books, songs, recipes — anything you can put a number on.' ),
```

Comment:
It would be great to have 3 `keywords` here. :-)

See example;
https://github.com/Automattic/jetpack/blob/d001db24166a884881e7d0a2f1c7e12429e053c6/extensions/blocks/tiled-gallery/index.js#L203-L207

Reply:
Why three? I can only think of `star` and `rating` as relevant.

Reply:
There is also `review`!

Reply:
added at https://github.com/Automattic/jetpack/pull/14091/commits/8c74d9db5b3f424dbe259533dc170564cda3b883

Reply:
3 is maximum so why not use them all. :-)

Reply:
Block inserter search picks terms from the block's name so you can still replace "star" and "rating" keywords with something else.

Some more ideas; "like", "reaction", "review", "rate", "rank", "vote", "grade"